monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -1916,6 +1917,34 @@ sealed abstract class Task[+A] extends TaskBinCompat[A] with Serializable {
     */
   final def uncancelable: Task[A] =
     TaskCancellation.uncancelable(this)
+
+  /**
+    * TODO: doc
+    *
+    * @return
+    */
+  final def timedAttempt: Task[(Duration, Either[Throwable, A])] =
+    Task.deferAction { scheduler =>
+      val now: Task[Long] = Task.eval(scheduler.clockMonotonic(TimeUnit.NANOSECONDS))
+
+      for {
+        start <- now
+        a     <- this.attempt
+        end   <- now
+      } yield (end - start).nanos -> a
+    }
+
+  /**
+    * TODO: doc
+    *
+    * @return
+    */
+  final def timed: Task[(Duration, A)] =
+    this
+      .timedAttempt
+      .flatMap { case (duration, attempted: Either[Throwable, A]) =>
```

Comment:
Why not implement `timed` on its own, and then `timedAttempt` would become just `.attempt.timed`?

Reply:
Because I'm stupid 😅I didn't see the possibility to do `.attempt.timed`.

I thinks that `timedAttempt` isn't useful so.